tsconfig.json

```
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES2020",
+    "target": "es2019",
```

I personally prefer capital ES, because this is the auto-suggested value when pressing ctrl+space in vscode 👀 
```suggestion
    "target": "ES2019",
```
But will this somehow affect our code? I mean, esbuild has its own target, so if I'm not wrong, this line doesn't affect anything 🤔